packages/amplication-client/src/Workspaces/ResourceList.tsx

```diff
@@ -112,16 +112,22 @@ function ResourceList() {
     [deleteResource, setError, trackEvent]
   );
 
-  const handleProjectDelete = useCallback(() => {
-    trackEvent({
-      eventName: AnalyticsEventNames.ProjectDelete,
-    });
-    deleteProject({
-      variables: {
-        projectId: currentProject.id,
-      },
-    }).catch(setError);
-  }, [currentProject, deleteProject, setError, trackEvent]);
+  const handleProjectDelete = useCallback(
+    (project) => {
```

Comment:
Is project being used anywhere?

Reply:
No, let me remove it and upload the video that I shared in Discord.